### ISSUE TITLE
iOS 7 fails to create database

### ISSUE THREAD

**Issue by dustinmoorenet** [axemclion/IndexedDBShim#121]
I am getting an error when I create a database only on iOS 7.x:
"Could not create table **sysdb** to save DB versions"

The error that returns off of onerror callback is:
SQLError
code: 5
message: "could not prepare statement (1 no such table: dbVersions)"

Here is the code I am running

```
<html>
<head>
<script type="text/javascript" src="js/IndexedDBShim.min.js"></script>
</head>
<body>
<div>upgrade:<span id="upgrade">working</span></div>
<div>success:<span id="ok">working</span></div>
<div>error:<span id="err">working</span></div>
<script>

var request = window.indexedDB.open('foo', 1);

request.onupgradeneeded = function() {
    document.getElementById('upgrade').innerHTML = 'done';
}

request.onsuccess = function(evt) {
    document.getElementById('ok').innerHTML = 'done';
}

request.onerror = function(evt) {
    console.log(evt);
    document.getElementById('err').innerHTML = 'done';
}
</script>
</body>
</html>
```

This code runs fine on iOS 5 as well as other OSes. I don't have iOS 6 to test on.

**Comment by DickvdBrink** [comment]
I noticed this issue recently. It has something to do with timing.
If you wait a few ms it works correctly. (which is kinda crappy I know).

You can find the reason behind it here: http://stackoverflow.com/questions/22684793/indexeddb-onupgradeneeded-not-firing-on-ios/22692216#22692216

**Comment by andygup** [comment]
I saw the exact same issues and this seems to have fixed'em: https://github.com/axemclion/IndexedDBShim/issues/144

**Comment by DickvdBrink** [comment]
Fixed with #148
Thanks for reporting
